mne/dipole.py

```diff
@@ -585,12 +593,12 @@ def plot(self, show=True, time_unit="s"):
 # IO
 @verbose
 def read_dipole(fname, verbose=None):
-    """Read ``.dip`` file from Neuromag/xfit or MNE.
+    """Read a dipole object from a file.
 
     Parameters
     ----------
     fname : path-like
-        The name of the ``.dip`` or ``.fif`` file.
+        The name of the ``.[b]dip`` (Neuromag/xfit) or ``.fif[.gz]`` (MNE) file.
```

Comment:
Technically Xfit can also produce DipoleFixed IIRC. Really it's that one is sequential fitting at each time point and the other is fixing a position (and maybe orientation) and estimating the currents to explain the data variance as well as possible

Reply:
So are you requesting that I delete the info in brackets for inaccuracy? :)

```suggestion
        The name of the ``.[b]dip`` or ``.fif[.gz]`` file.
```

☝️ like this?

Reply:
Yeah that would be fine